configs/sample_config.ini

```diff
@@ -0,0 +1,91 @@
+
+[DATASET]
+# Training image (.png) directory.
+TRAIN_DATA_DIR = data/charset_1k
+# Test image (.png) directory.
+TEST_DATA_DIR = data/charset_1k_test
+# Path to label file.
+LABEL_FILE = source/charset_1k.txt
+
+[IMAGE]
+# Image height.
+HEIGHT = 20
+# Image width.
+WIDTH = 20
+# Whether original image is in grayscale or RGB format.
+GRAYSCALE_IN = False
+# Whether to return grayscale image as output.
+GRAYSCALE_OUT = False
+
+[TRAIN_TEST_SPEC]
+# Training batch size.
+TRAIN_BATCH_SIZE = 64
+# Test batch size.
+TEST_BATCH_SIZE = 1
+# Higher: runs slower with better shuffling.
+SHUFFLE_BUFFER_SIZE = 200
+# Allows next n elements to be prepared.
+PREFETCH_BUFFER_SIZE = 20
+
+[DATA_AUG]
+# If true, randomly rotate image by a certain degree.
+RANDOM_ROTATE = True
+# Stddev of normal distribution (rotation angle in degree).
+ROTATE_STDDEV = 0.35
+
+# If true, randomly zoom in on the image by certain percent.
+RANDOM_ZOOM = True
+# Zoom in on the image by at most this percent.
+ZOOM_PERCENT = 4.0
+# Stddev of normal distribution (percentage to zoom in on).
```

Comment:
Change it to "StdDev of normal distribution to zoom-in the image. The unit of this value is the zoom in percentage."
I assume 0.4 here mean 0.4% of StdDev, right?

Reply:
Here it means the distribution of crop-in percentage follows a half normal distribution with 0 as mean and 0.4 as standard deviation. Meaning most of the images are unchanged or zoomed in by a very small percentage.

Reply:
So yes 0.4 here means 0.4% StdDev for zooming-in